invenio_requests/assets/semantic-ui/js/invenio_requests/timelineCommentEditor/state/actions.js

```diff
@@ -18,18 +18,44 @@ export const HAS_ERROR = "eventEditor/HAS_ERROR";
 export const SUCCESS = "eventEditor/SUCCESS";
 export const SETTING_CONTENT = "eventEditor/SETTING_CONTENT";
 
+const draftCommentKey = (requestId) => `draft-comment-${requestId}`;
```

Comment:
```suggestion
const draftCommentKey = (requestId) => `draft-comment-new-${requestId}`;
```
thinking ahead to differentiate with `reply` to a comment